lib/resolvers/PackageResolver.ts

```diff
@@ -105,6 +117,24 @@ export class PackageResolver {
     });
   }
 
+  /**
+   * Retrieves the latest Angular Material version remotely.
+   */
+  private static _retrieveLatestVersion(): Promise<string> {
+    return new Promise((resolve, reject) => {
+      request('http://material.angularjs.org/docs.json', (error, response, body) => {
```

Comment:
Well, that shouldn't be considered as hacky. It's the basic way to access things, which are not connected with Node.